Presentation.Web/Models/API/V1/ItSystemDTO.cs

```diff
@@ -65,6 +65,7 @@ public string ObjectOwnerFullName
         public int? InterfaceId { get; set; }
 
         public DateTime LastChanged { get; set; }
+        public string LastChangedByName { get; set; }
```

Comment:
LastChangedByUserName and LastChangedByUserLastName will be automatically mapped